install_config/cluster_metrics.adoc

```diff
@@ -268,6 +270,19 @@ xref:../install_config/cluster_metrics.adoc#metrics-reencrypting-route[re-encryp
 route] method.
 ====
 
+The Heapster component uses the service name DNS registry to connect to Hawkular Metrics.
+Update the URL used by Heapster to connect to Hawkular Metrics. For example, change:
+
+----
+--sink=hawkular:https://hawkular-metrics:443?tenant=_system&;....
+----
+
+To:
+
+----
+ --sink=hawkular:https://${CUSTOM_HOSTNAME}:443?tenant=_system&;....
+----
+
```

Comment:
--sink is hard coded in the deployer image as the heapster.yaml template that is called. 

https://github.com/openshift/origin-metrics/blob/master/deployer/templates/heapster.yaml#L83

This would need to be changed in the metrics code and should not be in the docs until the variable is added.